### ISSUE TITLE
Videos can't be played

### ISSUE THREAD

**Issue by markdimi** [mage2tv/issues#21]
Videos can't be played on my browser. Instead they display the following message:
"SORRY - Because of its privacy settings, this video cannot be played here"

Browser: Chrome 64.0.3282.140 64bit
OS: Opensuse Leap 42.2 
Kernel: 4.4.104-18.44

On Firefox there is no problem.

**Comment by Vinai** [comment]
Thanks for opening the issue here!
The video streaming platform vimeo allows embedding of content based on the host domain.
For this to work it relies on the headers of the browser.
Have you maybe configured Chrome to not send a `Referer` header?

For future reference, this is the beginning of the conversation:
https://twitter.com/Markulakos/status/970409962848604160

**Comment by dvdmierden** [comment]
I have the same problem, but on both browsers (Firefox & Chrome on MacOS)
Located in Netherlands.

The videos do play in Safari!

**Comment by Vinai** [comment]
Thanks for chiming in! Unfortunately I’m still unable to reproduce the issue...
Could you please post a screenshot of the message? Thanks!

**Comment by Vinai** [comment]
Are you by chance using some privacy enhancing browser extension, maybe one that hides the referrer header?

**Comment by dvdmierden** [comment]
D'oh! 
Silly me. That's it. I'm running Privacy Badger & uBlock Origin. After disabling both, the videos play on Firefox.

I tried again on Chrome. In fact I only needed to whitelist (allow) player.vimeo.com in Privacy Badger. Now it works!

Thanks for your quick reply! I was afraid I was stuck with Safari ;-)

**Comment by Vinai** [comment]
Thank you for that, this is good to know!
@dmarkos does the same apply to you?

**Comment by markdimi** [comment]
I will check and let you know!

Στις Πέμ, 31 Μαΐ 2018, 11:05 π.μ. ο χρήστης Vinai Kopp <
notifications@github.com> έγραψε:

**Comment by markdimi** [comment]
Yes. Privacy badger was the perpetrator. Although now the videos are subscription based, I can see the videos in the FAQ. Probably PB breaks Vimeo embedding.
